[doc] Considerations when deploying Bookkeeper (#13987)
apache/pulsar: site2/docs/administration-zk-bk.md

The minimum configuration changes required in `conf/bookkeeper.conf` are as follows:

> **Note**  

> > The configuration of journalDirectory and ledgerDirectories needs to be considered clearly, and it is very troublesome to change later.

Reviewer: I think it might be better to keep the description aligned across different files. Maybe considering use global replacement to change the description in all docs ? Thanks ~

The minimum configuration changes required in `conf/bookkeeper.conf` are as follows:

> **Note**

> > The configuration of journalDirectory and ledgerDirectories need to be chosen carefully，as it is very troublesome to change later.

Reviewer 2: ```suggestion
> Set `journalDirectory` and `ledgerDirectories` carefully. It is difficilt to change them later.
```

# Point to local ZK quorum
zkServers=zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181

> #This parameter best be set, otherwise，the BookKeeper can't start normally in special environment, for example, Huawei Cloud.

Reviewer 2: ```suggestion
# It is recommended to set this parameter. Otherwise, BookKeeper can't start normally in certain environments （for example, Huawei Cloud).
```
